<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_login_window</class>
 <widget class="QMainWindow" name="customer_login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customer Login Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 242, 248);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="csloginwdw_lbl_heading">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(15, 42, 57);</string>
    </property>
    <property name="text">
     <string>Customer Login </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="csloginwdw_btn_login">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(39, 82, 97);
background-color: rgb(211, 224, 243);
border:2px solid rgb(123, 169, 191);
border-radius:20px
}

QPushButton:hover {
     background-color: rgb(225, 229, 241);
     border:2px solid rgb(196, 218, 245);
}</string>
    </property>
    <property name="text">
     <string>  LOGIN</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/password.png</normaloff>icons/password.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="csloginwdw_btn_returnmain">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(39, 82, 97);
background-color: rgb(187, 207, 237);
border:2px solid rgb(123, 169, 191);
border-radius:20px
}

QPushButton:hover {
     background-color: rgb(225, 229, 241);
     border:2px solid rgb(196, 218, 245);
}</string>
    </property>
    <property name="text">
     <string>  Back</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/back-arrow.png</normaloff>icons/back-arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="csloginwdw_btn_exit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(39, 82, 97);
background-color: rgb(187, 207, 237);
border:2px solid rgb(123, 169, 191);
border-radius:20px
}

QPushButton:hover {
     background-color: rgb(225, 229, 241);
     border:2px solid rgb(196, 218, 245);
}</string>
    </property>
    <property name="text">
     <string>  Exit</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/log-out.png</normaloff>icons/log-out.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>252</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="csloginwdw_lbl_ADid">
       <property name="font">
        <font>
         <family>Bahnschrift SemiBold</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Customer ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="csloginwdw_linedit_ADid">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="csloginwdw_lbl_ADpassword">
       <property name="font">
        <font>
         <family>Bahnschrift SemiBold</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="csloginwdw_linedit_ADpassword">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="csloginwdw_lbl_warning">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 38, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
